tcpdf.php

```diff
@@ -13675,6 +13678,94 @@ protected function applyTSA($signature) {
 			return $signature;
 		}
 		//@TODO: implement this feature
+		// start timestamping
+		// by Hida since 5.9.128 (2011-10-06)
+		if($this->tsa_timestamp) {
+			//Include asn1 fuction script
+			require_once(dirname(__FILE__).'/include/asn1_parser_tcpdf.php');
+			require_once(dirname(__FILE__).'/include/asn1_function_tcpdf.php');
+			require_once(dirname(__FILE__).'/include/functionLog_tcpdf.php');
+
+			$tsaLog = __FILE__." line:(".__LINE__."). Perform timestamping...\n";
+			//Parse TCPDF Signature structure to get signed hash sequence
+			$p = asn1parse($signature);
+			$p1 = asn1parse($p[0][1]);
+			$p2 = asn1parse($p1[1][1]);
+			$p3 = asn1parse($p2[0][1]);
+			$p2 = asn1parse($p3[4][1]);
+			$pa1 = asn1parse($p2[0][1]);
+			$pa2 = asn1parse($pa1[3][1]);
+
+			//Create timestamp request
+
+			//Create hash of encrypted contents TCPDF signature
+			$hash = hash('sha1', hex2bin($pa1[5][1]));
+			//Build timestamp request data
+			$tsReqData = seq(
+											int(1).
+											seq(
+													seq(
+															"06052B0E03021A". // Obj_sha1
+															"0500" // Null
+															).
+													oct($hash)
+													).
+											int(hash('crc32', rand())).
+											'0101ff'
+											);
+			$raw_data = hex2bin($tsReqData);
+
+			//Send request to TSA Server with Curl
+			if(extension_loaded('curl')) {
+				$tsaLog .= __FILE__." line:(".__LINE__."). Curl was already Loaded\n".__FILE__." line:(".__LINE__."). Curl is sending tsRequest to \"".$this->tsa_data['tsa_host']."\" ...\n";
+				$ch = curl_init();
+				curl_setopt($ch, CURLOPT_URL, $this->tsa_data['tsa_host']);
+				curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
+				curl_setopt($ch, CURLOPT_POST, 1);
+				curl_setopt($ch, CURLOPT_HTTPHEADER, 
+					    	[
+						'Content-Type: application/timestamp-query',
+						'User-Agent: TCPDF'
+						]
```

Comment:
```suggestion
					    	array(
						'Content-Type: application/timestamp-query',
						'User-Agent: TCPDF'
						)
```
PHP 5.3 is still supported..